cmake/GolangSimple.cmake

```diff
@@ -10,15 +10,14 @@ function(ADD_GO_INSTALLABLE_PROGRAM NAME MAIN_SRC)
   get_filename_component(MAIN_SRC_ABS ${MAIN_SRC} ABSOLUTE)
   add_custom_target(${NAME})
   add_custom_command(TARGET ${NAME}
-                    COMMAND env GOPATH=${GOPATH} go build 
+                    COMMAND env GOPATH=${GOPATH} go build
                     -o "${CMAKE_CURRENT_BINARY_DIR}/${NAME}"
                     ${CMAKE_GO_FLAGS} ${MAIN_SRC}
                     WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
                     DEPENDS ${MAIN_SRC_ABS})
   foreach(DEP ${ARGN})
     add_dependencies(${NAME} ${DEP})
   endforeach()
-  
+
   add_custom_target(${NAME}_all ALL DEPENDS ${NAME})
-  install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/${NAME} DESTINATION bin)
```

Comment:
I would leave this line and then run 

     make DESTDIR=bin install  

then I'll adjust 
https://github.com/w3f/polkadot-re-tests/blob/0137463f3224a8150b29184e56eb00308987f922/test/test_fixtures.jl#L20
To point to the new correct location

    ../build/bin/usr/local/bin/scale_codec,
    ../build/bin/usr/local/bin/scale_codec_go

That's the standard behavoir of CMake with c++ which I'm gonig to add soon.